mods/ts/sequences/vehicles.yaml

```diff
@@ -104,7 +85,6 @@ gghunt:
 		Facings: 1
 		Length: 8
 		ShadowStart: 8
-	icon: seekicon
```

Comment:
the hunter-seeker should be a super-power and not buildable, so at least this removal looks right to me.